source/mainnet/net/guides/run-node-aws.rst

```diff
@@ -0,0 +1,165 @@
+.. _Discord: https://discord.gg/xWmQ5tp
+
+.. _run-a-node-aws:
+
+=================
+Run a node on AWS
+=================
+
+In this guide, you will learn how to setup a node on an AWS VPS, using the marketplace product.
+In this guide, the node will be running on the minimum required hardware, but you will be able to configure it to be running on more powerful hardware.
+
+
+Before you begin
+================
+
+Before continuing, you will need to have an AWS account.
+
+
+.. _running-a-node-aws:
+
+Running/upgrading a node
+========================
+
+Concordium provides two AWS Marketplace products, a `mainnet <https://aws.amazon.com/marketplace/pp/prodview-vfyzjfpphcciu?sr=0-1&ref_=beagle&applicationId=AWSMPContessa>`_ one and a `testnet <https://aws.amazon.com/marketplace/server/configuration?productId=prod-lumupqjfb6z6o&ref_=psb_cfg_continue>`_ one.
```

Comment:
**Question:** These links are asking me to login, do we have any public links? Maybe these are just not available yet.
The testnet one seems very different from the mainnet one, is it correct?

Reply:
Updated the links to look similar.
They are not public yet, they are under review from the AWS engineering team to become public